<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="RACIT061" failures="2" tests="2" name="Login test" time="18.097" errors="0" timestamp="2025-01-10T23:50:20 IST">
  <testcase classname="shoppingcart.test.LoginTest" name="submitOrderTest_updated" time="5.73">
    <failure type="java.lang.AssertionError" message="expected [THANKYOU FOR THE ORDER] but found [THANKYOU FOR THE ORDER.]">
      <![CDATA[java.lang.AssertionError: expected [THANKYOU FOR THE ORDER] but found [THANKYOU FOR THE ORDER.]
at shoppingcart.test.LoginTest.submitOrderTest_updated(LoginTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- submitOrderTest_updated -->
  <testcase classname="shoppingcart.test.LoginTest" name="submitOrderTest_updated" time="5.78">
    <failure type="java.lang.AssertionError" message="expected [THANKYOU FOR THE ORDER] but found [THANKYOU FOR THE ORDER.]">
      <![CDATA[java.lang.AssertionError: expected [THANKYOU FOR THE ORDER] but found [THANKYOU FOR THE ORDER.]
at shoppingcart.test.LoginTest.submitOrderTest_updated(LoginTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- submitOrderTest_updated -->
  <testcase classname="shoppingcart.test.LoginTest" name="submitOrderTest" time="0.0">
    <ignored/>
  </testcase> <!-- submitOrderTest -->
  <testcase classname="shoppingcart.test.LoginTest" name="orderHistoryTest" time="0.0">
    <ignored/>
  </testcase> <!-- orderHistoryTest -->
</testsuite> <!-- Login test -->
